bang/magicollections.py

```diff
@@ -1171,10 +1171,16 @@ def extra_context(self, context):
         'modal-text': 'true',
     },
 }
+
 EVENT_LIST_ITEM_CUTEFORM['status'] = {
     'type': CuteFormType.HTML,
 }
 
+EVENT_LIST_ITEM_CUTEFORM['i_boost_stat'] = {
+    'to_cuteform': lambda k, v: list(v)[0],
```

Comment:
you can grab the first char directly from the string, there's no need to list() it here